lib/cartopy/feature.py

```diff
@@ -43,6 +43,7 @@
 
 
 _NATURAL_EARTH_GEOM_CACHE = {}
+_GADM_GEOM_CACHE = {}
```

Comment:
The string on the following line is documentation for the variable on line 45, so you shouldn't insert this new one in between.